### PR TITLE
Add Speech v1

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

**Comment by michaelbausor** [comment]
Ping

File: generated/php/google-cloud-speech-v1/src/Speech/V1/resources/speech_client_config.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          "max_retry_delay_millis": 60000,
+          "initial_rpc_timeout_millis": 20000,
+          "rpc_timeout_multiplier": 1.0,
+          "max_rpc_timeout_millis": 20000,
+          "total_timeout_millis": 600000
+        }
+      },
+      "methods": {
+        "Recognize": {
+          "timeout_millis": 60000,
```
Comment (shinfan) [inline generated/php/google-cloud-speech-v1/src/Speech/V1/resources/speech_client_config.json]: We have updated this value to 190000 yesterday...

Comment (michaelbausor) [inline generated/php/google-cloud-speech-v1/src/Speech/V1/resources/speech_client_config.json]: Fixed

**Comment by michaelbausor** [comment]
PTAL

**Comment by shinfan** [comment]
lgtm

**Pull request merged**